<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>详细信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>668</width>
        <height>528</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
              <family>微软雅黑</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(68, 126, 217);
padding: 10px</string>
         </property>
         <property name="text">
          <string>详细信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="font">
          <font>
              <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>姓名</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>照片</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>编号</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>社会身份</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>职业</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>地区</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>户籍所在地</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>结婚登记日期</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="2">
          <property name="text">
           <string>所在社区/镇村</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>性别</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="2">
          <property name="text">
           <string>活跃度</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>出生日期</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="2">
          <property name="text">
           <string>大陆工作单位</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>学历</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="8" column="2">
          <property name="text">
           <string>地址</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_export">
           <property name="font">
            <font>
                <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_append">
           <property name="font">
            <font>
                <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_modify">
           <property name="font">
            <font>
                <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_delete">
           <property name="font">
            <font>
                <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="font">
            <font>
                <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
